cmd/commons/application_config.go

```diff
@@ -123,3 +142,29 @@ func updateApplicationConfig(configStructure interface{}, applicationName string
 
 	return ioutil.WriteFile(appCfgPath, configJSON, 0644)
 }
+
+func fillPasswordsFromKeyChain(configStructure *Config) error {
+	for serverName, coordinates := range configStructure.Servers {
+		srvPass, err := keyring.Get(ApplicationName, serverName)
+		if err != nil && err != keyring.ErrNotFound {
+			return err
+		}
+		if len(srvPass) > 0 && err != keyring.ErrNotFound {
+			Debugf("Pass for %s obtained from keychain successfully", serverName)
+			coordinates.Password = srvPass
+		}
+	}
+	return nil
+}
+
+func storePasswordsInKeyChain(configStructure *Config) error {
+	for serverName, coordinates := range configStructure.Servers {
+		err := keyring.Set(ApplicationName, serverName, coordinates.Password)
+		if err != nil {
+			return err
```

Comment:
if return here we can have some passwords set in keychain and some not. It might be better to just log the error and let rest of the credentials be stored to maximise what was successful

Reply:
Done